Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1479,6 +1479,8 @@ void MainWindow::readSettings()
           settings.value("default_sm_rm", "flat+edges").toString());
     CGAL::Three::Three::s_defaultPSRM = CGAL::Three::Three::modeFromName(
           settings.value("default_ps_rm", "points").toString());
+    // enable anti-aliasing
+    ui->actionAntiAliasing->setChecked(settings.value("antialiasing", false).toBool());
```

Comment:
Please try to compile at least the `Polyhedron_3` target. And fix the compilation errors. You cannot blindly fix conflicts without testing before the commit.

Reply:
For now I have tried with this PR merged in `integration`. Maybe the compilation error is in a conflict with `integration`...

Reply:
No: the compilation error is also in the PR alone!